src/tests/ftest/util/daos_utils.py

```diff
@@ -632,14 +632,15 @@ def object_query(self, pool, cont, oid, sys_name=None):
 
         return data
 
-    def filesystem_copy(self, src, dst):
+    def filesystem_copy(self, src, dst, preserve_props=None):
```

Comment:
This comment applies to several places in the python but shouldn't this be a True/False, what's the reason for using None here?

Reply:
@ashleypittman honestly I don't have a strong reason. I was recently learning how to navigate and add tests here so I followed what it looks like most of the other tests do, which is set the parameter to None when it is a non-required argument.

Reply:
`preserve_props` is actually a path it looks like. So I interpret `None` to mean "do not preserve" and `! None` to mean "do preserve, with this path"

Reply:
yeah it is a an option that when used takes a path. I suppose there could have been a true/false preserve_props option but it would still require passing in the path if the option was used.

Reply:
Ah, that makes sense.  None is the correct default then.